TARGETS

```diff
@@ -94,9 +94,6 @@ is_opt_mode = build_mode.startswith("opt")
 # -DNDEBUG is added by default in opt mode in fbcode. But adding it twice
 # doesn't harm and avoid forgetting to add it.
 ROCKSDB_COMPILER_FLAGS += (["-DNDEBUG"] if is_opt_mode else [])
-ROCKSDB_STRESS_DEPS = (
-        [":rocksdb_lib", ":rocksdb_test_lib"] if not is_opt_mode else [":rocksdb_lib"]
-)
```

Comment:
I don't quite get this but I just assume that TARET is generated from the script and everything works.

Reply:
The content in TARGETS is auto-generated. If we do not modify the buckifier scripts, these updates will be removed if we run `python buckifier/buckify_rocksdb` in the future releases.